Userland/Libraries/LibC/unistd.cpp

```diff
@@ -965,6 +966,7 @@ int pledge(char const* promises, char const* execpromises)
 int unveil(char const* path, char const* permissions)
 {
     Syscall::SC_unveil_params params {
+        static_cast<int>(UnveilFlags::CurrentProgram),
```

Comment:
similar question as the `jail_attach` PR, should we add a LibC wrapper for `unveil_after_exec` ?

Reply:
I don’t see a reason to… OpenBSD as far as I know doesn’t have this functionality of forcing veil attributes only after exec, and we don’t write our userland in pure C so there’s no way someone will use this very SerenityOS specific feature - it will be there just for the sake of “completeness” but without actual usage.